### PR TITLE
[ADD] Module sale_unified_menu

### PR DESCRIPTION
**Pull request by OpenCode**

### CODE REVIEW

**Comment by scigghia** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Better to call it _sale_order_unified_menu_

**Comment by OpenCode** [comment]
@pedrobaeza Why do you think it is better?

**Comment by pedrobaeza** [comment]
Because you are specifying in concrete what do you "unify"

**Comment by OpenCode** [comment]
@pedrobaeza done

File: sale_order_unified_menu/views/sale_view.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <data>
+
+        <record id="action_unified_order" model="ir.actions.act_window">
+            <field name="name">All Orders</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">sale.order</field>
+            <field name="view_type">form</field>
+            <field name="view_id" ref="sale.view_quotation_tree"/>
+            <field name="view_mode">tree,form,calendar,graph</field>
+            <field name="context">{'search_default_my_sale_orders_filter': 1}</field>
```
Comment (pedrobaeza) [inline sale_order_unified_menu/views/sale_view.xml]: I find this very annoying in the standard. Maybe we can omit this filter in this new menu.

Comment (OpenCode) [inline sale_order_unified_menu/views/sale_view.xml]: Context?

Comment (pedrobaeza) [inline sale_order_unified_menu/views/sale_view.xml]: This makes the filter "My" to be activated by default. And this filter is not useful most of the times.

Comment (OpenCode) [inline sale_order_unified_menu/views/sale_view.xml]: Ok

Comment (OpenCode) [inline sale_order_unified_menu/views/sale_view.xml]: @pedrobaeza done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by atchuthan** [comment]
:+1:

File: sale_order_unified_menu/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © 2015 - Francesco Apruzzese <f.apruzzese@apuliasoftware.it>
```
Comment (tafaRU) [inline sale_order_unified_menu/__init__.py]: Shouldn't be 2016? :wink:

File: sale_order_unified_menu/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 - Francesco Apruzzese <f.apruzzese@apuliasoftware.it>
```
Comment (tafaRU) [inline sale_order_unified_menu/__openerp__.py]: Same as above.

File: sale_order_unified_menu/__init__.py (hunk 2 above)
Comment (OpenCode) [inline sale_order_unified_menu/__init__.py]: @tafaRU Ehm..... Yes! Done!

File: sale_order_unified_menu/__openerp__.py (hunk 3 above)
Comment (OpenCode) [inline sale_order_unified_menu/__openerp__.py]: @tafaRU Done!

**Comment by tafaRU** [comment]
Thanks @OpenCode, I'm merging.

**Pull request merged**